apps/accounts/tests/test_provider_request.py

```diff
@@ -575,3 +594,42 @@ def test_approve_creates_evidence_documents(db):
     assert evidence_with_file.type == ev2.type
     assert evidence_with_file.valid_from == today
     assert evidence_with_file.valid_to == a_year_from_now
+
+
+@pytest.mark.django_db
+@override_flag("provider_request", active=True)
+def test_wizard_view_with_just_network_explanation(
+    user,
+    client,
+    wizard_form_org_details_data,
+    wizard_form_org_location_data,
+    wizard_form_services_data,
+    wizard_form_evidence_data,
+    wizard_form_network_explanation_only,
+    wizard_form_consent,
+    wizard_form_preview,
+):
+    client.force_login(user)
+
+    # given: valid form data and authenticated user
+    form_data = [
+        wizard_form_org_details_data,
+        wizard_form_org_location_data,
+        wizard_form_services_data,
+        wizard_form_evidence_data,
+        wizard_form_network_explanation_only,
+        wizard_form_consent,
+        wizard_form_preview,
+    ]
+    # when: a multi step submission has been successfully completed
+    response = _create_provider_request(client, form_data)
+
+    # then: a ProviderRequest object exists in the db
+    pr = response.context_data["providerrequest"]
+    assert models.ProviderRequest.objects.filter(id=pr.id).exists()
+
+    # then we have our explanation saved to the provider
```

Comment:
Small suggestion: if you want to check that the explanation is saved to the provider, then it makes sense to compare the expected value to the saved object (`models.ProviderRequest.objects.filter(id=pr.id).get()`) rather than the body of the HTTP response.